packages/front-end/components/Experiment/TabbedPage/index.tsx

```diff
@@ -197,6 +198,43 @@ export default function TabbedPage({
 
   const isBandit = experiment.type === "multi-armed-bandit";
   const trackSource = "tabbed-page";
+
+  const showMetricGroupPromo = (): boolean => {
+    // If the org has metric groups already & has the feature, don't show helper
+    // an org could not have the feature and have metric groups if they downgraded, this reminds them
+    // that they can get more feature if they upgrade
+    if (hasCommercialFeature("metric-groups") && metricGroups.length)
+      return false;
+
+    if (metricGroups.length) {
+      // if org has metric groups and metrics contain a metric group, don't show the helper
+      if (
+        experiment.goalMetrics.some((goalMetric) =>
+          isMetricGroupId(goalMetric)
+        ) ||
+        experiment.secondaryMetrics.some((secondaryMetric) =>
+          isMetricGroupId(secondaryMetric)
+        ) ||
+        experiment.guardrailMetrics.some((guardrailMetric) =>
+          isMetricGroupId(guardrailMetric)
+        )
+      ) {
+        return false;
+      }
+    }
```

Comment:
We can simplify both of these checks and just do `if (metricGroups.length) return false;`  We don't care about weird edge cases like someone had metric groups as enterprise, but then downgraded to free.